src/util.jl

```diff
@@ -39,10 +39,19 @@ end
 Base.replace(str::DenseStringViewAndSub, pat_repl::Pair{<:AbstractChar}; count::Integer=typemax(Int)) =
     replace(str, isequal(first(pat_repl)) => last(pat_repl); count=count)
 
-Base.replace(str::DenseStringViewAndSub, pat_repl::Pair{<:Union{Tuple{Vararg{<:AbstractChar}},
-                                            AbstractVector{<:AbstractChar},Set{<:AbstractChar}}};
-        count::Integer=typemax(Int)) =
+function Base.replace(
+    str::DenseStringViewAndSub,
+    pat_repl::Pair{
+        <:Union{
+            Tuple{Vararg{AbstractChar}},
```

Comment:
This indentation looks awfully strange. Maybe just fix the original issue without a reformat?

Reply:
IMO the new indentaiton is much more readable, but no worries, I'll change it back,